### PR TITLE
infra: Add window build test in CI

### PR DESCRIPTION
**Pull request by JSUYA**
Add build test for normal build

https://github.com/Samsung/thorvg/issues/600

### CODE REVIEW

File: src/lib/tvgCommon.h [hunk 1]
```diff
@@ -25,6 +25,10 @@
 #include "config.h"
 #include "thorvg.h"
 
+#ifdef _BUILD_WIN_
+#include <cstring>
```
Comment (hermet) [inline src/lib/tvgCommon.h:29]: I think we can isolate this include to the necessary files.
Could you please suggest the error position?

**Review by hermet — CHANGES_REQUESTED** [review]

File: src/lib/tvgCommon.h (hunk 1 above)
Comment (mgrudzinska) [reply src/lib/tvgCommon.h:29]: @hermet @JSUYA before I noticed this pr I prepared mine: https://github.com/Samsung/thorvg/pull/840

I dodn;t want to include string.h in tvgInitializer.cpp so I changed the code a bit.
I would appreciate your comment on such an approach.

Comment (JSUYA) [reply src/lib/tvgCommon.h]: I deleted the code change for the build fix.
I hope that build fail will be fixed in a separate patch(add cstring) or #840.

**Review by mgrudzinska — CHANGES_REQUESTED** [review]
cstring should be also included in the svg loader (tvgSvgLoaderCommon and tvgSvgUtil)

**Comment by mgrudzinska** [comment]
I've just got an error on windows using c++ (msys) compiler - it seams that string.h should be also included in tvgTvgSaver.cpp (strdup requires it)

File: src/lib/tvgInitializer.cpp [hunk 2]
```diff
@@ -23,6 +23,10 @@
 #include "tvgTaskScheduler.h"
 #include "tvgLoader.h"
 
+#ifdef _MSC_VER
```
[the remark this answers is not in the document]
Comment (JSUYA) [reply src/lib/tvgInitializer.cpp]: I changed to _WIN32

**Comment by JSUYA** [comment]
> I've just got an error on windows using c++ (msys) compiler - it seams that string.h should be also included in tvgTvgSaver.cpp (strdup requires it)

This PR is for normal build test in MSVC(`ilammy/msvc-dev-cmd@v1`) environment.
tvg Saver is not yet a default option.
I will consider it when adding the build test environment later.

**Comment by mgrudzinska** [comment]
@JSUYA the svg loader is the def opt and it requires cstring, but I understand it will be in a sep pr.

**Review by mgrudzinska — APPROVED** [review]

**Pull request merged**